sdk/src/loom/SQLite.ls

```diff
@@ -0,0 +1,423 @@
+/*
+===========================================================================
+Loom SDK
+Copyright 2011, 2012, 2013 , 2014, 2015
+The Game Engine Company, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+===========================================================================
+*/
+
+
+package loom.sqlite
+{
+    /**
```

Comment:
Are you referring specifically to the example usage comment? If so, cool,
I'll move 'er down.
On 17 Feb 2015 4:00 AM, "Ben Garney" <notifications@github.com> wrote:

> In sdk/src/loom/SQLite.ls
> <https://github.com/LoomSDK/LoomSDK/pull/261#discussion_r24787959>:
>
> > +You may obtain a copy of the License at
> > +
> > +   http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +===========================================================================
> > +*/
> > +
> > +
> > +package loom.sqlite
> > +{
> > +    /**
>
> I think this doc will get lost. Can we stick it on connection?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LoomSDK/LoomSDK/pull/261/files#r24787959>.
>